jest/src/test/java/io/searchbox/client/FailingProxyTest.java

```diff
@@ -0,0 +1,94 @@
+package io.searchbox.client;
```

Comment:
`io.searchbox.client.http` sounds more appropriate as it is *somewhat* testing the `JestHttpClient` class of the same package.